wp/plugin/README.md

```diff
@@ -9,6 +9,7 @@ Depends on:
     - bap 2.0.0-alpha
     - ounit 2.0.8
     - z3 4.8.6
+    - re 
```

Comment:
Is there a specific version for `re` we're using?

Reply:
Added 1.9.0